### PR TITLE
Use anchor instead of nextjs link to make use of the redirect on the index page

### PR DESCRIPTION
**Pull request by dabbeg**
## What

Using anchors instead of Link to make the browser refresh

## Why

Because that will trigger the redirect on the ingress to redirect all pages at loftbru.island.is -> island.is/loftbru
You can check the redirect out on https://loftbru.dev01.devland.is

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against master before asking for a review

### CODE REVIEW

File: apps/air-discount-scheme/web/components/Header/Header.tsx [hunk 1]
```diff
@@ -39,16 +36,13 @@ function Header({ routeKey, localeKey }: PropTypes) {
   const { toRoute, activeLocale, switchLanguage } = useI18n()
 
   const nextLanguage = activeLocale === 'is' ? 'en' : 'is'
+  const nextPath = activeLocale === 'is' ? '/' : '/en'
```
Comment (darrikonn) [inline apps/air-discount-scheme/web/components/Header/Header.tsx:39]: This should actually be `'is' ? 'https://island.is/loftbru' : 'https://island.is/en/lower-airfares-for-residents-in-rural-areas'`

**Review by darrikonn — CHANGES_REQUESTED** [review]

**Review by barabrian — APPROVED** [review]
Not sure what redirect is on the index page but this looks fine to me

File: apps/air-discount-scheme/web/components/Header/Header.tsx (hunk 1 above)
Comment (dabbeg) [reply apps/air-discount-scheme/web/components/Header/Header.tsx:39]: The redirect will handle that for us

Comment (darrikonn) [reply apps/air-discount-scheme/web/components/Header/Header.tsx:39]: aaaahhh sooooo smaaaaart

**Review by darrikonn — APPROVED** [review]

File: apps/air-discount-scheme/web/components/Header/Header.tsx (hunk 1 above)
Comment (darrikonn) [reply apps/air-discount-scheme/web/components/Header/Header.tsx:39]: What about local development?

**Pull request merged**